task_DEV-3.cs

```diff
@@ -0,0 +1,63 @@
+﻿using System;
+
+namespace FibonacciNumbers
+{
+    class Program
+    {
+        static void Main()
+        {
+            bool repeat = false;
+            const string reinput = "\nPlease, repeat number entry.\n";
+            int number = 0;
+            do
+            {
+                Console.WriteLine("Enter a number:");
+                string numberString = Console.ReadLine();                
+                try
+                {
+                    number = Int32.Parse(numberString);
+                    repeat = false;
+                }
+                catch (FormatException e)

```

Comment:
You have similar processing for all exceptions, try to combine it not to have similar code fragments. Do not repeat youself